File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewModelCustomTreeRowFieldRow.cs
/// <filename>
///     LowLevelFields_ViewModelCustomTreeRowFieldRow.cs
/// </filename>


namespace widgetassertions
{
    public class LowLevelFields_ViewModelCustomTreeRowFieldRow
  {
        private string HeaderLabelText;
     
        public     virtual string getHeaderLabelText()
    {
      return this.HeaderLabelText;
    }
     
        public int AdditionalInt;
     
        public     virtual int getAdditionalInt()
    {
      return this.AdditionalInt;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewModelCustomTreeRowFieldRow.cpp

#include "LowLevelFields_ViewModelCustomTreeRowFieldRow.h"
#include "LowLevelFields_ViewModelCustomTreeRowFieldRow.h"
#include <string>

namespace widgetassertions
{
  std::string LowLevelFields_ViewModelCustomTreeRowFieldRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  int LowLevelFields_ViewModelCustomTreeRowFieldRow::getAdditionalInt( ) 
  {
    return this->AdditionalInt;
  }
  std::string LowLevelFields_ViewModelCustomTreeRowFieldRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
  int LowLevelFields_ViewModelCustomTreeRowFieldRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

